### PR TITLE
Team Member Card View is completed.

### PR DESCRIPTION
**Pull request by Rishabh3321**
Card Design Link [https://codepen.io/fikirkod/pen/PoYVymv](url)
Modify/Add into teamMermberData for adding/editing more Members.

### CODE REVIEW

**Comment by Sharma-Hrishabh** [comment]
The cards are looking great. But it will be better if you could just shift the content of the about us page into the home page.
Keep the spacing in mind while moving the content in to the homepage :)

**Review by Sharma-Hrishabh — COMMENTED** [review]
On Hold

**Comment by Rishabh3321** [comment]
New changes 
- Content of About is now under Home Page.
- Line height and font size changes for increasing readability
- Join link underline and out of button clickable fixes.

Fonts, line-height, etc. are subjective. Suggestions will be welcome.

**Review by Sharma-Hrishabh — CHANGES_REQUESTED** [review]
LGTM except one thing. There should be more spacing between different sections viz. Landing, About and Team.
It will be good, if only one section is visible at once, and other appears after one scroll.

**Comment by Rishabh3321** [comment]
I have changed the height to the full viewport. But it is looking a little bit extra spacious. We should add more content to justify its full viewport or reduce the space to some scale.

File: src/portofolio_sections/AboutContent.jsx [hunk 1]
```diff
@@ -95,3 +102,7 @@ const P = styled.p`
   font-size: 20px;
   line-height: 2rem;
 `;
+
+const Frame = styled.div`
+  height: 100vh;
```
Comment (Ujjwal0501) [inline src/portofolio_sections/AboutContent.jsx:107]: making every division's height equal to the screen height is not a good decision.

It introduces unnecessary extra space.
image

File: src/portofolio_sections/AboutContent.jsx [hunk 2]
```diff
@@ -33,41 +33,48 @@ const teamMembersData = [
...
-        Research Paper Discussions 📰 <br />
-        Study Jams 🍟 x🍶 <br />
-        Talks Learn from the Experiences <br />
-        Blogs Pen it down… 🖊️ <br />
-        Open-Source Projects🖌️ 😄 <br />
-        Researches Closed Group 👥🌏 <br />
-        Open to all Developer Community (on Discord) 🕸️ <br />
-      </P>
+      <Frame>
+        <Col className="mt-2 p-0">
```
Comment (Ujjwal0501) [inline src/portofolio_sections/AboutContent.jsx:37]: Using `mt-#` and `p-#` classes is the better way to add spacing between div.

**Review by Ujjwal0501 — DISMISSED** [review]

**Comment by Ujjwal0501** [comment]
I think we can separately accept changes from commit [`dd5cb24` (#1)](https://github.com/BaNSC/bansc.github.io/pull/1/commits/dd5cb248744aa3189989c4073ca79bc41c3b3158) and a change for adding spacing in a way I suggested above.

Rishabh will have to pick that change and create a new PR. And also add the suggested spacing.

The changes related to Card View can be put into separate PR.

So I suggest the following:
1. Split this PR into two separate PRs
2. First PR will include changes from [`dd5cb24` (#1)](https://github.com/BaNSC/bansc.github.io/pull/1/commits/dd5cb248744aa3189989c4073ca79bc41c3b3158) + spacing related changes
3. Second PR will include changes from [`c567615` (#1)](https://github.com/BaNSC/bansc.github.io/pull/1/commits/c5676153a4554a1ac912e6a9988a74011e362b3e)
The card view will further need some changes so a separate PR is better.

**Comment by Sharma-Hrishabh** [comment]
I Agree @Ujjwal0501. The branch is deviating far from the current-branch. So, creating a separate PR will serve the purpose and we can merge the first PR, till you proceed with your work on the second one.

**Comment by Sharma-Hrishabh** [comment]
Closed via #3

**Pull request closed without merging**